src/cognito-verifier.ts

```diff
@@ -291,14 +314,24 @@ export class CognitoJwtVerifier<
   public verifySync<T extends SpecificVerifyProperties>(
     ...args: CognitoVerifyParameters<SpecificVerifyProperties>
   ): CognitoIdOrAccessTokenPayload<IssuerConfig, T> {
-    const payload = super.verifySync(...args);
-    const issuerConfig = this.getIssuerConfig(payload.iss);
-    const verifyProperties = {
-      ...issuerConfig,
-      ...args[1],
-    };
-    validateCognitoJwtFields(payload, verifyProperties);
-    return payload as CognitoIdOrAccessTokenPayload<IssuerConfig, T>;
+    const { decomposedJwt, jwksUri, verifyProperties } =
+      this.getVerifyParameters(args[0], args[1]);
```

Comment:
How`args` parameters are organized?

Reply:
Destructured them now for better readability. Does that solve your concern?